site/components/YoutubeEmbed/styles.ts

```diff
@@ -0,0 +1,52 @@
+import { css } from "@emotion/css";
+
+export const youtubeEmbed = css`
+  position: relative;
+  width: 100%;
+  border-radius: 16px;
+  overflow: hidden;
+
+  .youtubeEmbed_iframe {
+    border: none;
+    position: absolute;
+    width: 100%;
+    height: 100%;
+  }
+
+  .youtubeEmbed_posterBtn {
+    width: 100%;
+    height: 100%;
+    position: absolute;
+    top: 0;
+    left: 0;
+  }
+
+  .youtubeEmbed_playIconContainer {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+`;
+
+export const posterOverlay = (hidden: boolean) => css`
+  width: 100%;
+  opacity: ${hidden ? 0 : 1};
+  z-index: ${hidden ? -1 : "initial"};
```

Comment:
We prefer using pure CSS for dynamic styles.
So toggle a `.hidden` classname.

It's easiest with `cx` which is just a re-export of [classnames](https://github.com/JedWatson/classnames#usage)


EDIT: I added it to the styleguide here:
https://github.com/KlimaDAO/klimadao/wiki#prefer-plain-css

Reply:
Gotcha, updated